src/main/resources/io/jenkins/plugins/essentials/EssentialsManagementLink/index.jelly

```diff
@@ -0,0 +1,20 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:i="jelly:fmt" >
+  <l:layout norefresh="true" type="one-column" title="${%about(app.VERSION)}">
```

Comment:
This is not specifying a `permission`; make sure it is not visible to users without `ADMINISTER`. I cannot recall if we now default to checking that for any `ManagementLink` (CC @daniel-beck).